### PR TITLE
Add stuff

### PR DESCRIPTION
**Pull request by alejaa03**

### CODE REVIEW

File: staff/alejandro-perez/recursive-for/recursive-for.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+function recursiveFor(numbers,func) {
```
Comment (kimbali) [inline staff/alejandro-perez/recursive-for/recursive-for.js:1]: yep! ya saps que falta

File: staff/alejandro-perez/arrays-native/scripts/copyWithin.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    beforeEach(function () {
+        array = [1, 2, 3, 4, 5];
+    });
+
+    it('Copy within', function () {
+        var result = array.copyWithin(4,3,4);
+
+
+        expect(result.length === 5).toBeTruthy()
+        expect(result[4]).toBe(4)
```
Comment (jordiubanell) [inline staff/alejandro-perez/arrays-native/scripts/copyWithin.test.js:15]: semicolon everywere jajaj

File: staff/alejandro-perez/arrays-native/scripts/concat.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+describe('Array.prototype.concat', function () {
+    var array;
+    var array1;
+
+    beforeEach(function () {
+        array = ['a','b','c']
```
Comment (Javier1177) [inline staff/alejandro-perez/arrays-native/scripts/concat.test.js:8]: You missed semicolon! :(

File: staff/alejandro-perez/arrays-native/scripts/concat.test.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        array1 = ['d','e','f']
```
Comment (Javier1177) [inline staff/alejandro-perez/arrays-native/scripts/concat.test.js:9]: You missed semicolon again!

File: staff/alejandro-perez/arrays-native/scripts/fill.test.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+describe('Array.prototype.fill', function () {
+    var array;
+    beforeEach(function () {
+        array = [1, 2, 3, 4, 5]
+    });
+
+    it('Fills an array with only number', function () {
+        var result = array.fill(4);
+        expect(result).toEqual([4, 4, 4, 4, 4])
```
Comment (Javier1177) [inline staff/alejandro-perez/arrays-native/scripts/fill.test.js:11]: You missed semicolon!

File: staff/alejandro-perez/arrays-native/scripts/fill.test.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+    });
+
+    it('Fills an array', function () {
+        var result = array.fill(0,2,3);
+        expect(result).toEqual([1, 2, 0, 4, 5])
```
Comment (Javier1177) [inline staff/alejandro-perez/arrays-native/scripts/fill.test.js:17]: You missed semicolon again! :(

File: staff/alejandro-perez/restaurants-app/scripts/main.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
+// my custom components
+
```
Comment (jordiubanell) [inline staff/alejandro-perez/restaurants-app/scripts/main.js:2]: use strict man!! :)

File: staff/alejandro-perez/restaurants-app/scripts/main.js [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+SearchPanel.prototype = Object.create(Component.prototype);
+SearchPanel.prototype.constructor = SearchPanel;
+
+
+function ResultsList(tag, filteredArray) {
+    Component.call(this, 'div');
+    // ID DIV
+    this.element.id = 'resultsContainer'
```
Comment (Javier1177) [inline staff/alejandro-perez/restaurants-app/scripts/main.js:30]: You missed semicolon!

File: staff/alejandro-perez/restaurants-app/scripts/main.js [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+    var myList = document.createElement("ul")
```
Comment (Javier1177) [inline staff/alejandro-perez/restaurants-app/scripts/main.js:31]: You missed semicolon again! :_(

File: staff/alejandro-perez/restaurants-app/scripts/main.js [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+    myList.setAttribute("id", "resultsForm")
```
Comment (Javier1177) [inline staff/alejandro-perez/restaurants-app/scripts/main.js:32]: You missed semicolon!

File: staff/alejandro-perez/arrays-native/scripts/concat.test.js (hunk 3 above)
Comment (SergioCastilloSorribes) [reply staff/alejandro-perez/arrays-native/scripts/concat.test.js:8]: He thinks that works in EcmaScript 6....

**Pull request merged**